src/run-command.js

```diff
@@ -2,7 +2,7 @@ import execa from 'execa';
 import redisDownload from 'redis-download';
 import { homedir } from 'os';
 
-export default async function runCommand(command, args) {
+export default async function runCommand(command, args, opts) {
```

Comment:
```suggestion
export default async function runCommand(command, args, { stdio = 'inherit', detached = false } = {}) {
```

This way you don't have to sprinkle `{ studio: 'inherit' }` everywhere.